tools/check-lint-version/Readme.md

```diff
@@ -0,0 +1,12 @@
+# Check Lint Version
```

Comment:
Could you add this tool to the existing build tools in `.build-tools`? Could be another command in the same binary.